### ISSUE TITLE
Add Ahmad as a speaker

### ISSUE THREAD

**Issue by ChristinFrohne** [gitpod-io/devxconf.org#95]
Bio:
https://ahmadawais.com/speaker-bio/

Role:
Head of Developer Relations @RapidAPI

https://twitter.com/MrAhmadAwais
https://www.linkedin.com/in/mrahmadawais/

profile picture:
![ahmad-awais-dp](https://user-images.githubusercontent.com/46345125/115667896-a834c180-a346-11eb-820f-984137105e7f.jpeg)

**Comment by ChristinFrohne** [comment]
Abstract: After 10 years with Sublime Text, I switched to VSCode — the new open-source cross-platform editor everyone's talking about. I've spent over 1,000 hours perfecting my setup to help you switch today. In this talk, you'll get to learn tens of VSCode Power User workflows that'll help you save time and code more efficiently. Text manipulation to multi-cursors and even Markdown tips/tricks. You name it.

Title: VSCode Power User Tips & Tricks
